src/components/Layout/Sidebar/index.tsx

```diff
@@ -237,9 +239,11 @@ const Sidebar: React.FC<SidebarProps> = ({ open, setClosed }) => {
             <div className="flex flex-col flex-1 pt-5 pb-4 overflow-y-auto">
               <div className="flex items-center flex-shrink-0 px-4">
                 <span className="text-2xl text-gray-50">
-                  <a href="/">
-                    <img src="/logo.png" alt="Logo" />
-                  </a>
+                  <Link href="/">
+                    <a>
+                      <img src="/logo.png" alt="Logo" />
+                    </a>
+                  </Link>
```

Comment:
Same thing here. Not mobile but serves the same purpose.

Reply:
Alright, I'll change it back.